networking/enable-cluster-wide-proxy.adoc

```diff
@@ -14,7 +14,7 @@ The cluster-wide proxy is only supported if you used a user-provisioned infrastr
 
 == Prerequisites
 
-* Review the xref:../installing/install_config/configuring-firewall.adoc#configuring-firewall[sites that your cluster requires access to] and determine whether any of them must bypass the proxy. By default, all cluster egress traffic is proxied, including calls to the cloud provider API for the cloud that hosts your cluster. Add sites to the Proxy object's `spec.noProxy` field to bypass the proxy if necessary.
+* Review the xref:../installing/install_config/configuring-firewall.adoc#configuring-firewall[sites that your cluster requires access to] and determine whether any of them must bypass the proxy. By default, all cluster system egress traffic is proxied, including calls to the cloud provider API for the cloud that hosts your cluster. (System-wide proxy affects system components only, not user workloads.) Add sites to the Proxy object's `spec.noProxy` field to bypass the proxy if necessary.
```

Comment:
We try to avoid parentheticals, so I would remove the parentheses. Other than that, looks great!